tests/lib/tools/suite/fs-state/task.yaml

```diff
@@ -0,0 +1,45 @@
+summary: smoke test for the fs test tool
+execute: |
+    # Check help
+    "$TESTSTOOLS"/fs-state -h | MATCH "usage: fs-state wait-for-file"
+    "$TESTSTOOLS"/fs-state --help | MATCH "usage: fs-state wait-for-file"
+
+    # Check non existing directory
+    "$TESTSTOOLS"/fs-state ensure-dir-exists "$PWD"/mytestdir
+    test -d "$PWD"/mytestdir
+    test -f "$PWD"/mytestdir.fake
+
+    "$TESTSTOOLS"/fs-state clean-dir "$PWD"/mytestdir
+    not test -d "$PWD"/mytestdir
+
+    # Check existing directory
+    mkdir "$PWD"/mytestdir
+    "$TESTSTOOLS"/fs-state ensure_dir_exists_backup_real "$PWD"/mytestdir
```

Comment:
Please test the version with `-` instead of `_`.

Reply:
done